src/stylesheets/elements/_buttons.scss

```diff
@@ -1,181 +1,215 @@
 .usa-button {
-  @include u-margin-y("1px");
-  @include u-text('secondary-darker');
-  @include u-padding-y(1.5);
-  @include u-padding-x(4);
+  @include u-text("secondary-darker");
+  @include u-border("primary");
+  @include u-border-width("2px");
 
   &.usa-button--hover,
   &:hover {
-    @include u-bg('primary-light');
-    @include u-text('secondary-darker');
+    @include u-bg("primary-light");
+    @include u-border("primary-light");
+    @include u-border-width("2px");
+    @include u-text("secondary-darker");
   }
 
   &.usa-button--active,
   &:active {
-    @include u-bg('secondary-darker');
-    @include u-text('white');
+    @include u-bg("secondary-darker");
+    @include u-border("secondary-darker");
+    @include u-text("white");
   }
 
   &.usa-focus,
   &:focus {
     outline-offset: units(2px);
   }
+  &--disabled,
```

Comment:
you only need :disabled not a separate class

Reply:
This is how USWDS does it

Reply:
No, they only use the :disabled selector...
<img width="1680" alt="Screen Shot 2020-06-18 at 7 04 46 AM" src="https://user-images.githubusercontent.com/63304927/85030209-0777e300-b132-11ea-8849-33ac8cb4bd82.png">